packages/plugin-core/src/constants.ts

```diff
@@ -902,6 +902,12 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     },
     when: "dendron:pluginActive",
   },
+  LOCK_PREVIEW: {
+    key: "dendron.lockPreview",
+    title: `${CMD_PREFIX} Lock Preview`,
+    icon: `$(open-preview)`,
+    when: "dendron:pluginActive",
+  },
```

Comment:
FYI: I did not add shortkeys.

Reply:
thats fine - we've been avoiding them for new commands